fastlane_core/lib/fastlane_core/update_checker/changelog.rb

```diff
@@ -3,7 +3,7 @@
 module FastlaneCore
   class Changelog
     class << self
-      def show_changes(gem_name, current_version)
+      def show_changes(gem_name, current_version, update_command: "bundle update")
```

Comment:
This was to break the circular dependency between `changelog.rb` and `update_checker.rb`

Reply:
Alternative is to move the `UpdateChecker.update_command` method into a new class `UpdateCheckerUtils` and make both class call it. That way, no API change.

If you keep it like you proposed, maybe rename the parameter `update_gem_command` ?

Reply:
Hmm, I think I'll keep it, but do the rename. I saw there's a bunch of tests associated with the method as well, which would require more refactoring. From a usage perspective the API doesn't change anyway, because the parameter has an optional default value